executables/collector/src/test/java/de/cyface/collector/AuthorisationTest.java

```diff
@@ -64,26 +70,29 @@
     void test(final Vertx vertx, final VertxTestContext ctx) throws URISyntaxException, IOException {
         final var publicKey = Files.readString(Path.of(this.getClass().getResource("/public.pem").toURI()));
         final var privateKey = Files.readString(Path.of(this.getClass().getResource("/private_key.pem").toURI()));
-        final var keyOptions = new PubSecKeyOptions().setAlgorithm(RS256).setBuffer(publicKey)
-                .setBuffer(privateKey);
-        final var options = new JWTAuthOptions();
+
+        final var provider = JWTAuth.create(vertx, new JWTAuthOptions()
+                .addPubSecKey(new PubSecKeyOptions()
+                        .setAlgorithm(JWT_HASH_ALGORITHM)
+                        .setBuffer(publicKey))
+                .addPubSecKey(new PubSecKeyOptions()
+                        .setAlgorithm(JWT_HASH_ALGORITHM)
+                        .setBuffer(privateKey)));
+
         final var issuer = "localhost:8080/api/v2/";
+        final var body = new JsonObject().put("username", "test").put("password", "test-password")
+                .put("aud", issuer).put("iss", issuer);
+        final var jwtOptions = new JWTOptions()
+                .setAlgorithm(JWT_HASH_ALGORITHM).setIssuer(issuer).setAudience(Collections.singletonList(issuer));
+        final var token = provider.generateToken(body, jwtOptions);
```

Comment:
Before we set the JWTOptions in provider.setJwtOptions, which did not work after changing to RS256.
I got multiple errors when executing the Test.

When I set the JWTOptions in `generateToken()` as suggested in the Vert.X Dokumentation everything works with RS256, too.